src/libraries/System.Private.CoreLib/Tools/GenUnicodeProp/Program.cs

```diff
@@ -175,6 +175,7 @@ private static void PrintSourceIndexArray(string tableName, DataTable d, StreamW
             PrintByteArray(tableName.Replace('1', '3'), file, levels[2]);
         }
 
+#pragma warning disable 8714
```

Comment:
Instead of suppressing this, can you add `where T : notnull` as a constraint to the method?

Reply:
I'll try that.